examples/uplink_set.rb

```diff
@@ -0,0 +1,56 @@
+# (c) Copyright 2016 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+my_client = {
+  url: '',
+  user: '',
+  password: ''
+}
+
+# Example: creates or updates an uplink set

```

Comment:
Maybe everything is self-explanatory (I don't know much about uplink sets), but the more comments here the better. Volumes are pretty simple, but check out the comments on those examples: https://github.com/HewlettPackard/oneview-chef/blob/master/examples/volume.rb#L18